src/DynamoCoreWpf/Properties/Resources.Designer.cs

```diff
@@ -3925,6 +3889,78 @@ public class Resources {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Delete.
+        /// </summary>
+        public static string PackageContextMenuDeletePackageText {
+            get {
+                return ResourceManager.GetString("PackageContextMenuDeletePackageText", resourceCulture);
+            }
+        }
+        
+        /// <summary>
+        ///   Looks up a localized string similar to Deleting this package is only allowed if none of the types are currently in use.
```

Comment:
My impression was that this tooltip message referred to actually removing from disk (not scheduling or marking).
But we could update the message. Any ideas ? FYI @Jingyi-Wen 
Maybe - "Removes the package from Dynamo. Might require a restart for changes to take effect"..
Something similar for unloading ?

Reply:
Changing the message makes sense. Can we possibly avoid using "might" though? It's important to indicate what would happen for certain

Reply:
**Unload tooltip:**
Unloads this package so that it will not be visible in Dynamo anymore. The package will still exist in the Dynamo packages locations.

**Delete tooltip:**
Deletes this package from the Dynamo package locations. Once deleted, it will not be visible in Dynamo anymore.

Reply:
@Jingyi-Wen @aparajit-pratap would that work ?

Reply:
Make sense to me!

Reply:
Looks fine to me. @Amoursol ?

Reply:
LGTM team! Thanks for the loop in @aparajit-pratap

Reply:
@pinzart90 by "visible" do you mean visible in the node library? Or visible in the installed packages view? (my understanding is that unloaded pacakges will still show up in the packages view - but with an "unloaded" tag)

Reply:
@mjkkirschner  I was thinking about the node library.
Should I just be specific about the node library ? That may be a bit complicated for regular users ... ?

Reply:
Or should I just say:

**Unload tooltip:**
Unload this package so that it will not be loaded into Dynamo anymore. The package will still exist in the Dynamo packages locations.

**Delete tooltip:**
Delete this package from the Dynamo package locations. Once deleted, it will not be visible in Dynamo anymore.

Reply:
I'm a little late to the conversation - so please disregard if you are happy with the current state.
but for your consideration:

**Unload tooltip:**
Unload this package so that it will not be loaded into Dynamo. The package will not be deleted from disk.

**Delete tooltip:**
Delete this package from disk. Once deleted, it will not be loaded into Dynamo anymore until it is re-downloaded.